base/datafmt.jl

```diff
@@ -277,8 +273,6 @@ function readdlm_string(sbuff::ByteString, dlm::Char, T::Type, eol::Char, auto::
         catch ex
             if isa(ex, TypeError) && (ex.func == :store_cell)
                 T = ex.expected
-            elseif get(optsd, :ignore_invalid_chars, false)
```

Comment:
OT: Judging by the frequent complaints about performance we get, I also think these functions should be deprecated in favor of CSV.jl as soon as it's ready. CSV.jl+DataStreams.jl will also unify reading data into different structures like `Array`, `DataFrame`, `TimeSeries`, data bases, etc.